src/org/labkey/test/tests/SampleSetParentColumnTest.java

```diff
@@ -692,42 +691,32 @@ public void testAliasNameConflictsWithFieldName()
 
         log("Add a parent alias column to the sample set that conflicts with a given column name.");
         sampleHelper.goToEditSampleSet(SAMPLE_SET_NAME);
-
         sampleHelper.addParentColumnAlias(Map.of(ALIAS_NAME_CONFLICT, CURRENT_SAMPLE_SET_OPTION));
-
         clickButton("Update");
 
         Locator errorMsgLocator = Locator.tagWithClass("div", "labkey-error");
-
         Assert.assertTrue("Error message is not present.", isElementPresent(errorMsgLocator));
         Assert.assertTrue("Error message is not visible.", isElementVisible(errorMsgLocator));
         Assert.assertEquals("Error message not as expected.", "A sample set property already exists with parent alias header: " + ALIAS_NAME_CONFLICT, errorMsgLocator.findElement(getDriver()).getText());
 
         log("Now add a valid parent column and check that you cannot now add a field in the sample set with the same name.");
-
         sampleHelper.addParentColumnAlias(Map.of(GOOD_PARENT_NAME, CURRENT_SAMPLE_SET_OPTION));
-
         clickButton("Update");
 
-        clickButton("Edit Fields");
-
-        PropertiesEditor fieldProperties = new PropertiesEditor.PropertiesEditorFinder(getWrappedDriver()).withTitle("Field Properties").waitFor();
+        clickFolder(SUB_FOLDER_NAME);
+        DomainFormPanel domainFormPanel = sampleHelper.goToEditSampleSetFields(SAMPLE_SET_NAME);
+        domainFormPanel.addField(GOOD_PARENT_NAME);
+        clickButton("Save", 0);
 
-        fieldProperties.addField();
+        errorMsgLocator = Locator.tagWithClass("div", "alert-danger");
+        waitForElement(errorMsgLocator);
+        String errorMsgExpectedTxt = "'" + GOOD_PARENT_NAME + "' is a reserved field name in '" + SAMPLE_SET_NAME + "'.";
+        Assert.assertTrue("Error message not as expected.", errorMsgLocator.findElement(getDriver()).getText().contains(errorMsgExpectedTxt));
```

Comment:
Use hamcrest assertion to generate better error messages.
```suggestion
        Assert.assertThat("Error message", errorMsgLocator.findElement(getDriver()).getText(), CoreMatchers.containsString(errorMsgExpectedTxt));
```

Reply:
Done.